src/main/groovy/io/github/cosmicsilence/scalafix/ScalafixExtension.groovy

```diff
@@ -68,4 +75,23 @@ class ScalafixExtension {
     void setConfigFile(String path) {
         configFile.set(project.file(path))
     }
+
+    /**
+     * Checks if SemanticDB is enabled in the extension. It will return true as
+     * long as neither autoConfigureSemanticdb nor semanticdb.autoConfigure are false.
+     */
+    boolean isSemanticdbEnabled() {
+        autoConfigureSemanticdb != false && semanticdb.autoConfigure.get() != false
+    }
+
+    @Nested
+    @Optional
+    SemanticdbParameters getSemanticdb() {
+        return semanticdb
+    }
+
+    void semanticdb(Action<? super SemanticdbParameters> action) {
+        action.execute(semanticdb);
+    }
```

Comment:
newbie question: what is this for?

Reply:
This is basically using the closure `semanticdb` to set the values of the SemanticdbParameters object that has been instantiated in the constructor.

I am just replacing it by the following (seems easier to understand):
```
void semanticdb(Closure closure) {
    project.configure(semanticdb, closure)
}
```